server/sonar-server/src/main/java/org/sonar/server/user/index/UserIndexer.java

```diff
@@ -58,52 +61,104 @@ public void indexOnStartup(Set<IndexType> emptyIndexTypes) {
     doIndex(newBulkIndexer(Size.LARGE), null);
   }
 
+  public void commitAndIndex(DbSession dbSession, UserDto user) {
+    EsQueueDto queueItem = dbClient.esQueueDao().insert(dbSession, EsQueueDto.create(EsQueueDto.Type.USER, user.getLogin()));
+    dbSession.commit();
+    postCommit(dbSession, user, queueItem);
+  }
+
+  /**
+   * Entry point for Byteman tests. See directory tests/resilience.
+   * The unused parameter "user" is used only by the Byteman script.
+   */
+  private void postCommit(DbSession dbSession, UserDto user, EsQueueDto queueItem) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "user". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)